### ISSUE TITLE
import.xml doesn't allow numbers in vendor name and class name

### ISSUE THREAD

**Issue by tzsolti** [magento/magento2#4470]
## Preconditions

1. Magento 2.

## Steps to reproduce
1. Create a simple module with a vendor name which contains at least one number (ex. test123) which creates an import entity
2. etc/import.xml will look something like:

```
<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_ImportExport:etc/import.xsd">
<entity name="catalog_product" label="Products test" model="test123\somename\Model\Something" behaviorModel="Magento\ImportExport\Model\Source\Import\Behavior\Basic" />
</config>
```
3. Enable module, clear the cache and open the Magento backend Import page

## Expected result
1. It should load the import page

## Actual result
1. The site will die with an error message similar to this:

Invalid XML in file <path_to_website_root>/app/code/test123/somename/etc/import.xml:
Element 'entity', attribute 'model': [facet 'pattern'] The value 'test123\somename\Model\Something' is not accepted by the pattern '[A-Za-z_\\]+'.
Line: 3

Element 'entity', attribute 'model': 'test123\somename\Model\Something' is not a valid value of the atomic type 'modelName'.
Line: 3

The same happens also for export.xml.

**Comment by maksek** [comment]
@tzsolti, thanks for reporting, we are aware of the issue, and going to resolve it.

**Comment by nhp** [comment]
Would be fixed by PR #5420

**Comment by magento-engcom-team** [comment]
@tzsolti, thank you for your report.
The issue is already fixed in develop branch, 2.2.1
